build.sbt

```diff
@@ -378,6 +378,17 @@ lazy val `kyo-examples` =
             libraryDependencies += "com.softwaremill.sttp.tapir" %% "tapir-json-zio" % "1.11.7"
         )
 
+lazy val `kyo-tests` =
```

Comment:
Open to better naming.

Reply:
Could we put the new tests in `kyo-core` instead?